tests/functional/z_cluster/test_must_gather_minimal_crds.py

```diff
@@ -0,0 +1,152 @@
+                    pytest.mark.polarion_id("OCS-6312"),
+                    skipif_external_mode,
+                    skipif_ms_consumer,
+                    skipif_hci_client,
+                    tier2,
+                ],
+            ),
+            pytest.param(
+                *[False, True, False, False, False, False],
+                marks=[
+                    pytest.mark.polarion_id("OCS-6313"),
+                    skipif_external_mode,
+                    skipif_ms_consumer,
+                    skipif_hci_client,
+                    tier2,
+                ],
+            ),
+            pytest.param(
+                *[False, False, True, False, False, False],
+                marks=[
+                    pytest.mark.polarion_id("OCS-6314"),
+                    skipif_external_mode,
+                    skipif_ms_consumer,
+                    skipif_hci_client,
+                    tier2,
+                ],
+            ),
+            pytest.param(
+                *[False, False, False, True, False, False],
+                marks=[
+                    pytest.mark.polarion_id("OCS-6315"),
+                    skipif_external_mode,
+                    skipif_ms_consumer,
+                    skipif_hci_client,
+                    tier2,
+                ],
+            ),
+            pytest.param(
+                *[False, False, False, False, True, False],
+                marks=[
+                    pytest.mark.polarion_id("OCS-6311"),
+                    skipif_external_mode,
+                    skipif_ms_consumer,
+                    skipif_hci_client,
+                    tier2,
+                ],
+            ),
+            pytest.param(
+                *[False, False, False, False, False, True],
+                marks=[
+                    pytest.mark.polarion_id("OCS-6310"),
+                    skipif_external_mode,
+                    skipif_ms_consumer,
+                    skipif_hci_client,
+                    tier2,
+                    stretchcluster_required_skipif,
+                ],
+            ),
+        ],
+    )
+    def test_must_gather_minimal_crd_modular(
+        self, ceph, ceph_logs, namespaced, clusterscoped, noobaa, dr
+    ):
+        """
+        Tests OCS must gather with --minimal-crd flag and modular flags
+
+        Test Process:
+        1.Collect mg with relevant flags for example
+            oc adm must-gather --image=quay.io/rhceph-dev/ocs-must-gather:latest-4.15 -- /usr/bin/gather -c -cl -n
+        2.Calculate the paths that should be included in the ocs mg dir
+        3.Calculate the paths that should not be included in the ocs mg dir
+        4.Verify paths exist in must gather directory
+        5.Verify paths do not exist in must gather directory
+        """
+
+        flags_cmd = "/usr/bin/gather --minimal "
+        paths_exist = list()
+        paths_not_exist = list()
+        for path in const_must_gather.MINIMAL:
+            paths_exist.append(path)
```

Comment:
do we need to iterate here?
```suggestion
        paths_exist += const_must_gather.MINIMAL
```

will do that in one line I think, no?